XCTestHTMLReport/XCTestHTMLReport
# switch relative to absolute url during downsizing

#275 fixes relative attachment URLs

However, when running the downsizeImages step (with `-z` flag), the relative URL is passed to the converter which ultimately fails.

To fix this, we'll reconstruct the absolute URL when converting the image. Afterwards, a new _relative_ URL will be returned pointing to the new downsized image.

To test:
Run `xchtmlreport -r path/to/xcresult -z` from a **different working directory**, verify heic screenshots are converted to jpeg and properly displayed in the report.
